x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/detection_alerts/enrichments/alert_threat_enrichments.cy.ts

```diff
@@ -71,39 +69,37 @@ describe('Threat Match Enrichment', { tags: ['@ess', '@serverless', '@skipInServ
     });
   });
 
-  it('Displays persisted enrichments on the JSON view', () => {
```

Comment:
The addition of the `Table tab` test is good, but could you explain why the JSON view coverage was removed from cypress?

Reply:
hey @rylnd, I can explain for sure! I tried for 2+ hours to fix the JSON test, but the issue is the new flyout uses the `JsonCodeEditor` from the `unified_doc_viewer` which is different from the old flyout that was using a simple `EuiCodeBlock`. The code block was allowing us to easily retrieve the content and test against it. The `JsonCodeEditor` does not allow to do that.
The only way I kinda found to get it to work was to Cypress scroll to the correct position and select the row, but this is extremely flaky and we should never do that.

My thought was, for this test we don't really care if the value is showing in the JSON tab. The purpose of this test is not to verify that the flyout JSON tab shows the correct value. We should trust that the JSON tab and its components are rendering the document correctly (and we already have tests for that on the expandable flyout section). What we care about if the fact that the document has the correct value. The easiest way I found to do this was by filtering the table and verifying that the expected value was there.

Is that a wrong assumption?